pkg/operational_test_resources.go

```diff
@@ -46,8 +45,8 @@ func FetchOrganizationID(testAccProvider *schema.Provider) (string, error) {
 // FetchProjectID will find the first project given an organization and retrieve its ID.
 func FetchProjectID(orgID string, testAccProvider *schema.Provider) (string, error) {
 	// Initialize Client with connection settings
-	if err := testAccProvider.Configure(terraform.NewResourceConfigRaw(nil)); err != nil {
-		return "", err
+	if err := testAccProvider.Configure(context.Background(), terraform.NewResourceConfigRaw(nil)); err != nil {
```

Comment:
what about adding `ctx context.Context` to call, and leave it to called to decide which ctx to provide instead of creating backround ctx here?